saleor/graphql/core/types.py

```diff
@@ -103,6 +99,11 @@ class Meta:
         description = 'Represents a permission object in a friendly form.'
 
 
+class SeoInput(graphene.InputObjectType):
+    seo_title = graphene.String(description='SEO title.')
```

Comment:
I think `title` and `description` would be fine, we can drop the `seo_` prefix here.